### DISCUSSION TITLE
Killed

### DISCUSSION THREAD

**Discussion by pavelgolikov** [pyg-team/pytorch_geometric#6895] [Q&A]
May I please know what this means? Is it possible to get a more detailed error message?

![image](https://user-images.githubusercontent.com/40681739/224417909-d75a2a7b-ec12-4db3-bb16-c2eb22f9d9f1.png)

**Comment by EdisonLeeeee** [comment]
Since ogbn-products is a large dataset, you might have insufficient memory for the script.
